v03_pipeline/lib/vep.py

```diff
@@ -1,20 +1,29 @@
 import hail as hl
 
-from v03_pipeline.lib.model import DatasetType
+from v03_pipeline.lib.model import DatasetType, ReferenceGenome
+
+
+def validate_vep_config_reference_genome(reference_genome, config: str) -> None:
+    with open(config) as f:
+        if reference_genome.value not in f.read():
+            msg = f'Vep config does not match supplied reference genome {reference_genome.value}'
+            raise ValueError(msg)
 
 
 def run_vep(
     ht: hl.Table,
     dataset_type: DatasetType,
+    reference_genome: ReferenceGenome,
     vep_config_json_path: str | None,
 ) -> hl.Table:
     if not dataset_type.veppable:
         return ht
     config = (
         vep_config_json_path
         if vep_config_json_path is not None
-        else 'file:///vep_data/vep-gcloud.json'
+        else f'file:///vep_data/vep-{reference_genome.value}-gcloud.json'
```

Comment:
where does the vep config json live?

Reply:
It's basically created for us by hail's initialization of the VEP environment on dataproc.  Our local installation provides a local VEP config set [here](https://github.com/broadinstitute/seqr-loading-pipelines/tree/main/docker/vep_configs), which actually already has the reference genome embedded in the path.